core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -78,6 +78,8 @@
   UNDERFS_S3_THREADS_MAX(Name.UNDERFS_S3_THREADS_MAX, 22),
   UNDERFS_S3_UPLOAD_THREADS_MAX(Name.UNDERFS_S3_UPLOAD_THREADS_MAX, 2),
   UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS(Name.UNDERFS_S3A_CONSISTENCY_TIMEOUT_MS, 60000),
+  UNDERFS_S3A_REGION(Name.UNDERFS_S3A_REGION, "us-east-1"),
```

Comment:
Previously, this did not exist. Therefore, shouldn't the default be null or empty?